<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #22FF00;</string>
   </property>
   <property name="text">
    <string>Время:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #22FF00;</string>
   </property>
   <property name="text">
    <string>Мелодия:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_hour">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #22FF00;</string>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_minute">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>30</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #22FF00;</string>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_second">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #22FF00;</string>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QPushButton" name="select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color: #22FF00;
    background-color: #052200;}
QPushButton:hover{
    border: 1px solid #22FF00;
    background-color: #072800;}
QPushButton:pressed{
    color: black;
    background-color: #22FF00;}</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_track_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #22FF00;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set_alarm">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color: #22FF00;
    background-color: #052200;}
QPushButton:hover{
    border: 1px solid #22FF00;
    background-color: #072800;}
QPushButton:pressed{
    color: black;
    background-color: #22FF00;}</string>
   </property>
   <property name="text">
    <string>Установить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
